providers/oidc.go

```diff
@@ -176,6 +168,52 @@ func (p *OIDCProvider) createSessionState(token *oauth2.Token, idToken *oidc.IDT
 	return newSession, nil
 }
 
+func (p *OIDCProvider) CreateSessionStateFromBearerToken(rawIDToken string, idToken *oidc.IDToken) (*sessions.SessionState, error) {
+	newSession, err := p.createSessionStateInternal(rawIDToken, idToken, nil)
+	if err != nil {
+		return nil, err
+	}
+
+	newSession.AccessToken = rawIDToken
```

Comment:
Why do we use `rawIDToken` here?

Reply:
What else should it be? This method replaces the code that was in `GetJwtSession` so it does the same thing as it did, here: https://github.com/oauth2-proxy/oauth2-proxy/blob/c7bfbdecef0a724b5bc752607edd4257928ddd40/oauthproxy.go#L1122

You know the code far better than I do, what would you have used?

Reply:
Ack, I see that now, all is good 👍